grading/ssh-sudoer/bin/run_student

```diff
@@ -0,0 +1,32 @@
+#! /bin/python3
```

Comment:
Same remark that for ``bin/INGInious`` here: we won't maintain two copies with slight changes. If the behaviour differs, it would be better to define hooks and call them from the main ``run_student`` script.